sound/soc/sof/intel/pci-ptl.c

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: (GPL-2.0-only OR BSD-3-Clause)
+//
+// This file is provided under a dual BSD/GPLv2 license.  When using or
```

Comment:
Didnt we decide to remove all this @plbossart

Reply:
Good question. I don't remember if we made a conscious decision to remove this, but some files in the SOF tree don't have this comment. I'll keep it for now to keep the alignment with pci-lnl.c.